client/tests/qunit/tests/workflow_editor_tests.js

```diff
@@ -1023,6 +1022,15 @@ QUnit.module("terminal mapping logic", {
     verifyNotMappedOver: function (assert, terminal) {
         assert.ok(!terminal.mapOver.isCollection);
     },
+    verifyDefaultMapOver: function(assert, terminal) {
+        const outputCollectionTerminal = this.newOutputCollectionTerminal("list");
+        assert.ok(!terminal.node.mapOver);
+        const connector = new Connector({}, outputCollectionTerminal, terminal);
+        outputCollectionTerminal.connect(connector);
```

Comment:
That's the opposite of the situation in the bug report, right (connecting a list to a parameter input, in the bug report it's connecting a parameter output to a collection input) ? I think that was already working correctly prior to the fix.

Reply:
The issue with the connection described in the bug report is that the mapping from a collection output to the input parameter does not propagate the collection mapping state to the input parameter. With this fix the input parameter is recognized as mapped input and thus allows to continue the mapping in subsequent connections. See here:
<img width="901" alt="image" src="https://user-images.githubusercontent.com/2105447/150146840-df50cac0-5634-4708-a406-407c53f14b32.png">